packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Need to merge main so we can get the updated getIntervalId that can't be undefined.